client/browser/src/libs/bitbucket/code_intelligence.tsx

```diff
@@ -115,11 +125,35 @@ function getCommandPaletteMount(): HTMLElement {
     return document.querySelector<HTMLElement>(commandListClasses.map(c => `.${c}`).join('')) || createCommandList()
 }
 
+function getViewContextOnSourcegraphMount(): HTMLElement | null {
+    const branchSelectorButtons = document.querySelector('.branch-selector-toolbar .aui-buttons')
+    if (!branchSelectorButtons) {
+        return null
+    }
+    const preexisting = branchSelectorButtons.querySelector('#open-on-sourcegraph') as HTMLElement | null
+    if (preexisting) {
+        return preexisting
+    }
+    const mount = document.createElement('span')
+    mount.id = 'open-on-sourcegraph'
+    mount.className = 'open-on-sourcegraph__bitbucket-server'
```

Comment:
This is a modifier in BEM
```suggestion
    mount.className = 'open-on-sourcegraph--bitbucket-server'
```